### PR TITLE
Use LDAP lookup for user account email domain

### PR DESCRIPTION
**Pull request by aploshay**
Will require a new release of the `ldap_groups_lookup` gem to fully take effect, and subsequent bundle update, to fully take effect.

@dlpierce I didn't see that there was already a 0.5.0 branch in `ldap_groups_lookup`; how do you want to resolve that with the pr I just merged there?

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -77,7 +77,7 @@ gem 'devise-i18n'
 gem 'devise-guests', '~> 0.6'
 gem 'omniauth'
 gem 'omniauth-cas'
-gem 'ldap_groups_lookup', '~> 0.4.0'
+gem 'ldap_groups_lookup', '~> 0.5.0'
```
Comment (dlpierce) [inline Gemfile]:
```suggestion
gem 'ldap_groups_lookup', '~> 0.6.0'
```
I bumped the upstream gem version.

**Comment by aploshay** [comment]
@dlpierce Huh, do the CircleCI build isn't picking up on the new gem version?

**Comment by dlpierce** [comment]
Needed to actually push the .gem. Still failing though.

**Comment by aploshay** [comment]
@dlpierce I incorporated your requested changes, but then I force pushed squashed commits so I don't think it's tracking

**Pull request merged**